### ISSUE TITLE
[PostgreSQL] Table comment is missing from DDL.

### ISSUE THREAD

**Issue by cpburnz** [dbeaver/dbeaver#8679]
#### System information: 
- Operating system (distribution) and version: Ubuntu 18.04 64-bit
- DBeaver version: 7.0.4.202005031953
- Additional extensions: None

#### Connection specification:
- Database name and version: PostgreSQL 9.1 and 9.4.7
- Driver name: PostgreSQL JDBC Driver
- Do you use tunnels or proxies (SSH, SOCKS, etc)?  No

#### Describe the problem you're observing:

The DDL for the table comment is not displayed under the DDL tab even when "Show comments" is checked.

#### Steps to reproduce, if exist:

Create a table with a comment:

```sql
CREATE TABLE test_table (
	name text
);

COMMENT ON TABLE test_table IS 'Some description.';
```

The DDL for the table will be shown as:

```sql
-- Drop table

-- DROP TABLE public.test_table;

CREATE TABLE public.test_table (
	"name" text NULL
);

-- Permissions

ALTER TABLE public.test_table OWNER TO postgres;
GRANT ALL ON TABLE public.test_table TO postgres;
```

The comment is missing even when "Show comments" is checked. There should be a line for the comment as:

```sql
COMMENT ON TABLE public.test_table IS 'Some description.';
```

**Comment by uslss** [comment]
thanks for the bug report

**Comment by uslss** [comment]
fixed in scope of #8686
